shell/server/drivers/external-ui-view.js

```diff
@@ -64,7 +64,7 @@ function getOAuthServiceInfo(url) {
       service: "google",
       endpoint: "https://www.googleapis.com/oauth2/v4/token",
     };
-  } else if (url.startsWith("https://api.github.com/users")) {
+  } else if (url.startsWith("https://api.github.com/")) {
```

Comment:
After merging and then reading other PRs I realize that you probably need to change this so that it also accepts `https://api.github.com` (with no trailing `/`). But it shouldn't accept `https://api.github.com.evil.com`, of course.